### PR TITLE
Modified navbar styling.

### PR DESCRIPTION
**Pull request by brandondykun**
Modified the nav bar to better match the footer design.

### CODE REVIEW

File: app/components/Navbar.tsx [hunk 1]
```diff
@@ -14,29 +14,66 @@ import {
...
+    (currentRoute === "/" && href === "/#about")
+  );
+};
+
+type HeaderNavLinkProps = {
+  href: string;
+  text: string;
+};
+
+const HeaderNavLink = ({ href, text }: HeaderNavLinkProps) => {
```
Comment (ghost) [inline app/components/Navbar.tsx]: I'm going to push back a little here with this one... Is the amount of difference between `HeaderNavLink` and `MobileNavLink` enough to justify the reuse of code? I understand the desire to split them into two separate sub-components, but things like `currentRoute`, `isActiveRoute`, and even the base `Link` structure is the same between the two components.

I would advise this instead, even just for the sake of practicing.

Make a generic `NavLink` component. Props are `href`, `text`, `isMobile`, and `onClick?`. `isMobile` is a boolean used within `NavLink` to determine which `className` to use in the `Link` and determines whether or not to use the `onClick` `div`.

With this design we avoid repeating code, while still maintaining the ability to have modular design and functionality. If we thought more than two types of `NavLink`s could exist we would make `isMobile` into `type` and use a switch statement inside the `NavLink` component.

Hopefully that all makes sense!

**Review by ghost — CHANGES_REQUESTED** [review]
Looks great and matches the footer perfectly!

Main need for revision is the code structuring in `Navbar.tsx`. Let me know if you have any questions!

**Comment by brandondykun** [comment]
Link components are now refactored into one component. The typing was a little tricky, so if you know a better way to do that I'd be interested in hearing it. The idea is that isMobile and onClick need to be passed together and if either one is passed without the other it will show an error. Now that I write that, that might be redundant. But in our current design, if it's a mobile link, it needs the onClick to close the mobile menu since that doesn't happen automatically. If it's a desktop nav button, there isn't a foreseeable need to attach an onClick, so the typing doesn't allow it. Is this the best way to handle that?

**Review by ghost — APPROVED** [review]
Thank you for revising the `NavLink` code. Looks perfect!

**Pull request merged**